interfaces/builtin/mount_control.go

```diff
@@ -125,6 +125,7 @@ var allowedFilesystemSpecificMountOptions = map[string][]string{
 	"jfs":        {"iocharset=", "resize=", "nointegrity", "integrity", "errors=", "noquota", "quota", "usrquota", "grpquota"},
 	"msdos":      {"blocksize=", "uid=", "gid=", "umask=", "dmask=", "fmask=", "allow_utime=", "check=", "codepage=", "conv=", "cvf_format=", "cvf_option", "debug", "discard", "dos1xfloppy", "errors=", "fat=", "iocharset=", "nfs=", "tz=", "time_offset=", "quiet", "rodir", "showexec", "sys_immutable", "flush", "usefree", "dots", "nodots", "dotsOK="},
 	"nfs":        {"nfsvers=", "vers=", "soft", "hard", "softreval", "nosoftreval", "intr", "nointr", "timeo=", "retrans=", "rsize=", "wsize=", "ac", "noac", "acregmin=", "acregmax=", "acdirmin=", "acdirmax=", "actimeo=", "bg", "fg", "nconnect=", "max_connect=", "rdirplus", "nordirplus", "retry=", "sec=", "sharecache", "nosharecache", "revsport", "norevsport", "lookupcache=", "fsc", "nofsc", "sloppy", "proto=", "udp", "tcp", "rdma", "port=", "mountport=", "mountproto=", "mounthost=", "mountvers=", "namlen=", "lock", "nolock", "cto", "nocto", "acl", "noacl", "local_lock=", "minorversion=", "clientaddr=", "migration", "nomigration"},
+	// TODO: drop nfs4?
```

Comment:
I think so, from what I gather `nfs` defaults to 4 since nfs-utils 1.2 (15 years ago) and kernels 2.6.33. It's also deprecated from use in /etc/fstab. From manpages https://www.man7.org/linux/man-pages/man5/nfs.5.html when no nfsvers is specified, the client will start with 4.2 and then negotiate down until a compatible version is found (3 being the lowest anyway).

Reply:
Also:
```
maciek@galeon:~ ls -l `which mount.nfs4`
lrwxrwxrwx 1 root root 9 10-20 09:17 /usr/bin/mount.nfs4 -> mount.nfs
```

I'll just proceed to drop nfs4 from mount-control. There are no users, since nobody was able to specify the right attributes anyway.

Reply:
Agreed